.github/Feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Please put this file under `./github/ISSUE_TEMPLATE/` folder

Reply:
Done!